audio/midi_drivers/FluidSynthMidiDriver.cpp

```diff
@@ -64,13 +68,22 @@ int FluidSynthMidiDriver::open() {
 	if (!soundfont.empty())
 		soundfonts.push_back(soundfont);
 
+	_settings = new_fluid_settings();
+
 	if (soundfonts.empty()) {
-		perr << "FluidSynth requires a 'fluidsynth_soundfont' setting" << std::endl;
-		return -2;
+		char *default_soundfont;
+		if (getStr("synth.default-soundfont", &default_soundfont) == FLUID_OK &&
+		    default_soundfont && default_soundfont[0] != 0) {
+			soundfonts.push_back(default_soundfont);
+			fluid_free(default_soundfont);
+			perr << "FluidSynth without a 'fluidsynth_soundfont' setting, using default soundfont" << std::endl;
+		}
+		else {
+			perr << "FluidSynth has not default soundfont, requires a 'fluidsynth_soundfont' setting" << std::endl;
```

Comment:
The grammar in this message is very weird. If I am understanding it correctly, please change it to something like

`Fluidsynth does not have a default soundfont configured, and there was no `fluidsynth_soundfont' setting in Exult.cfg. Fluidsynth will not be enabled.`